tests/net/lib/dns_sd/src/main.c

```diff
@@ -521,7 +519,7 @@ static void test_dns_sd_handle_ptr_query(void)
 	struct in_addr addr = {
 		.s_addr = htonl(IP_ADDR(177, 5, 240, 13)),
 	};
-	static uint8_t actual_rsp[512];
+	static uint8_t actual_rsp[512] = {};
```

Comment:
Adding `{}` is not needed as the array is already cleared to 0 (because it is static)